styles.css

```diff
@@ -0,0 +1,93 @@
+@media (min-width: 375px) {
+  .toolbar-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 375px;
+    height: 48px;
+    padding: 0 12px;
+  }
+
+  #logo {
+    width: 76px;
+    height: 20px;
+    margin-top: 10px;
+  }
+
+  #nav-logo {
+    width: 16px;
+    height: 20px;
+  }
+
+  #headline {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 375px;
```

Comment:
> Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels

- [ ] According to the project requirement, it suggests you use percentages instead of px for your **widths**, this helps make your portfolio adapt to different screen sizes, In this case, this will make your headline section adapt to multiple screens sizes easily, and look professional and give the best user experience. :smile: :wave:

Reply:
@V-Blaze Could you please explain this part. Thanks